test/config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
we already have https://github.com/paulirish/pwmetrics/blob/master/test/fixtures/run-expectations.js

This file is not needed.